toolkit/tools/imagecustomizerschemacli/Makefile

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+BINARY_NAME := bin/imagecustomizerschemacli
+GO_FILES := $(wildcard *.go)
+SCHEMA_FILE := schema.json
+
+all: build run
+
+build: $(GO_FILES)
+	mkdir -p bin
+	go build -o $(BINARY_NAME) .
+
+run: build
+	@echo "Running $(BINARY_NAME) with $(SCHEMA_FILE)"
+	./$(BINARY_NAME) $(SCHEMA_FILE)
+
+clean:
+	rm -rf bin
+
+.PHONY: all build run clean
```

Comment:
I think it is better to have separate `.PHONY` lines for each command, so that the `.PHONY` can be placed in the same location as the command itself. This makes it a lot easier to see that a command is a `.PHONY`.